src/IO/GIPL_ImageFormat.cxx

```diff
@@ -129,7 +130,7 @@ Image::~Image()
 void
 Image::GiplRead(char* filename)
 {
-  printf("Read %s\n", filename);
+  fmt::print("Read {}", filename);
```

Comment:
would we have to use `internal_format::print` for these types of statements?

Reply:
good point, yes we should